src/ContMechTensors.jl

```diff
@@ -17,7 +17,7 @@ immutable InternalError <: Exception end
 
 export AbstractTensor, SymmetricTensor, Tensor, Vec, FourthOrderTensor, SecondOrderTensor
 
-export otimes, otimes_unsym, ⊗, ⊡, dcontract, dev, vol
+export otimes, otimes_unsym, ⊗, ⊡, dcontract, dev, vol, symmetric, skew
 export extract_components, load_components!, symmetrize, symmetrize!
```

Comment:
can you remove `symmetrize` and symmetrize!` while you are going. They are not in the package anymore